### PR TITLE
VZ-5396: Renaming secrets to be more backup specific

### PR DESCRIPTION
**Pull request by theabmitra**
Renaming secrets to be more backup specific

### CODE REVIEW

File: pkg/constants/constants.go [hunk 1]
```diff
@@ -230,9 +230,9 @@ const PrometheusClusterNameLabel = "verrazzano_cluster"
 
 const (
 	// Constants required for updating Opensearch keystore
-	VerrazzanoSecretName          = "verrazzano"
+	VerrazzanoKubeObjectName      = "verrazzano-backup"
```
Comment (kminder) [inline pkg/constants/constants.go:233]: This seems a poor variable name.  Does there need to be two variables?

Comment (theabmitra) [reply pkg/constants/constants.go:233]: We cannot have the word "Secret" in a variable lint. thats flaged by golint

Comment (pfmackin) [reply pkg/constants/constants.go:233]: Can you use xyzScrt or something.

Comment (kminder) [reply pkg/constants/constants.go:233]: I was thinking it needed to have Backup in it somewhere.  This is a fairly broadly shared constants file right?

Comment (theabmitra) [reply pkg/constants/constants.go]: yes, updated the secret name to have satisfy both the criteria. safe name and backup

**Review by kminder — APPROVED** [review]
Approved.

**Pull request merged**